pkg/util/crypto.go

```diff
@@ -38,7 +38,7 @@ import (
 // The certificate will be created with file mode 0644. The key will be created with file mode 0600.
 // If the certificate or key files already exist, they will be overwritten.
 // Any parent directories of the certPath or keyPath will be created as needed with file mode 0755.
-func GenerateSelfSignedCert(host, certPath, keyPath string) error {
+func GenerateSelfSignedCert(host, certPath, keyPath string, ServiceReadWriteIP net.IP) error {
```

Comment:
This function isn't just used by the apiserver. It's also used by the kubelet. By not passing in the Subject alternate names, the self signed certs used by the kubelet will now also have 'kubernetes.default.svc', 'kubernetes.default', and 'kubernetes' on the cert, which isn't ideal.